ipa-core/src/protocol/ipa_prf/prf_sharding/mod.rs

```diff
@@ -1068,4 +1071,54 @@ pub mod tests {
             assert_eq!(result, &expected);
         });
     }
+
+    #[test]
+    fn capping_bugfix() {
+        run(|| async move {
+            let world = TestWorld::default();
+
+            let records: Vec<PreShardedAndSortedOPRFTestInput<BA8, BA3, BA20>> = vec![
+                /* First User */
+                oprf_test_input_with_timestamp(10251308645, false, 218, 0, 29135),
+                oprf_test_input_with_timestamp(10251308645, true, 0, 3, 179334),
```

Comment:
It would be more helpful if the timestamps were ordered.

Reply:
What do you mean? They are in increasing time order.

Reply:
Oops, digit count miss on my end :(

Reply:
Clippy made me add some underscores, that should improve readability.